docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc

```diff
@@ -77,25 +67,25 @@ openstack overcloud node provision \
 ----
 +
 When the update is complete, ensure that the {CephCluster} nodes have a
-new interface created and tagged with the appropriate VLAN associated with
+new interface created and tagged with the VLAN that is associated with
 `StorageNFS`.
 
 . Identify an IP address from the `StorageNFS` network to use as the Virtual IP
-address for the Ceph NFS service. This IP address must be provided in place of
-the `{{ VIP }}` in the example below. You can query used IP addresses with:
+address for the Ceph NFS service. You must provide this IP address in place of
+the `{{ VIP }}` in the example below:
 +
 ----
-openstack port list -c "Fixed IP Addresses" --network storage_nfs
+$ openstack port list -c "Fixed IP Addresses" --network storage_nfs
 ----
 +
 * Pick an appropriate size for the NFS cluster. The NFS service provides
 active/active high availability when the cluster size is more than
 one node. It is recommended that the ``{{ cluster_size }}`` is at least one
-less than the number of hosts identified. This solution has been well tested
+less than the number of hosts identified. This solution has been tested
 with a 3-node NFS cluster.
-* The `ingress-mode` argument must be set to ``haproxy-protocol``. No other
-ingress-mode will be supported. This ingress mode will allow enforcing client
-restrictions through {rhos_component_storage_file}.
+* You must set the `ingress-mode` argument to `haproxy-protocol`. No other
+ingress-mode is supported. This ingress mode allows you to enforce client
+restrictions through the {rhos_component_storage_file}.
```

Comment:
I’m confused by the placement of this list. The information about picking an appropriate size of the cluster and setting the ingress-mode sounds like it belongs in the intro or prerequisites.

Reply:
I think we should remove `{{ cluster_size }}` and decide a size in advance. In theory this is driven by the placement "by label" (nfs is the label here), so we do not need to specify the cluster size.
I propose to remove the following:
```
* Pick an appropriate size for the NFS cluster. The NFS service provides
active/active high availability when the cluster size is more than
one node. It is recommended that the ``{{ cluster_size }}`` is at least one
less than the number of hosts identified. This solution has been tested
with a 3-node NFS cluster.
```